### ISSUE TITLE
Jquery Angular conflict Solution Example	

### ISSUE THREAD

**Issue by pakainfo-com** [angular-ui/ui-sortable#553]
I resove this good Articles but how?

http://pakainfo.com/jquery-angular-conflict-solution-example/
--

**Comment by thgreasi** [comment]
Hi, I don't really get what you mean. Can you rephrase?
Also, in the article I couldn't see any relevant code and the code areas are looking like this:
![image](https://user-images.githubusercontent.com/1295829/58321228-43a11d00-7e26-11e9-9e9d-6026aedbb893.png)
